cmd/notary/keys.go

```diff
@@ -130,6 +132,8 @@ func (k *keyCommander) GetCommand() *cobra.Command {
 		nil,
 		"New key(s) to rotate to. If not specified, one will be generated.",
 	)
+	cmdRotateKey.Flags().StringSliceVar(&k.rootCert, "rootcert", nil, "Public key used in key rotation")
```

Comment:
@cyc115: I think it's fine as-is, since we can only have `# public keys <= # private keys` since we cannot generate private keys corresponding to an existing public key?  See David's [previous comment](https://github.com/docker/notary/pull/1147/files/1974573c7499e00a04f20ef20269c1a641dfb16e#r122272437) - please let me know if I'm getting it backwards!